### PR TITLE
i9n: fix outpoint_assets and asset_allocations responses

### PR DESCRIPTION
**Pull request by zoedberg**
this PR fixes changes of #100

also it adds a fix to build worklow to avoid 404 errors when installing dependencies

### CODE REVIEW

File: src/i9n/fungible.rs [hunk 1]
```diff
@@ -191,25 +191,28 @@ impl Runtime {
...
-    ) -> Result<(), Error> {
+    ) -> Result<Reply, Error> {
         match &*self.command(Request::Allocations(contract_id))? {
             Reply::Failure(failure) => Err(Error::Reply(failure.clone())),
-            Reply::Success => {
+            Reply::Allocations(response) => {
                 println!("Asset allocations succeeded");
 
-                Ok(())
+                Ok(reply::Reply::Allocations(response.clone()))
```
Comment (dr-orlovsky) [inline src/i9n/fungible.rs:200]: No need to wrap object into Reply type here and below. And the return type of the fn can be changed as well

**Review by dr-orlovsky — CHANGES_REQUESTED** [review]
Thank you for catching apt problem!

Pls see the comment regarding the return type

File: src/i9n/fungible.rs (hunk 1 above)
Comment (zoedberg) [reply src/i9n/fungible.rs:200]: I don't get what do you mean, if I remove object wrapping like this
```
Ok(response.clone())
```
it gives me this error:
```
    |                    expected enum `reply::Reply`, found struct `BTreeMap`
    |                    help: try using a variant of the expected enum: `reply::Reply::Allocations(response.clone())`
```
I'm sorry if it's a stupid question but I'm still learning rust. moreover what should be the return type if not `Reply`?

Comment (zoedberg) [reply src/i9n/fungible.rs:200]: Ok sorry, the question was stupid. The function should return a BTreeMap object, right?

**Review by dr-orlovsky — APPROVED** [review]

**Pull request merged**